README.md

```diff
@@ -59,16 +59,18 @@ Thanks!
 ## Capabilities
 
 **Development Environment for Data**
-* **Experimentation** - try tools, upgrade versions and evaluate code changes in isolation. 
+* **Experimentation** - try tools, upgrade versions, and evaluate code changes in isolation. 
 * **Reproducibility** - go back to any point of time to a consistent version of your data lake.
 
 **Continuous Data Integration**
 * **Ingest new data safely by enforcing best practices** - make sure new data sources adhere to your lake’s best practices such as format and schema enforcement, naming convention, etc.  
 * **Metadata validation** - prevent breaking changes from entering the production data environment.
 
 **Continuous Data Deployment**
-* **Instantly revert changes to data** - if low quality data is exposed to your consumers, you can revert instantly to a former, consistent and correct snapshot of your data lake.
-* **Enforce cross collection consistency** - provide to consumers several collections of data that must be synchronized, in one atomic, revertible, action.
+
+* **Instantly revert changes to data** - if low quality data is exposed to your consumers, you can revert instantly to a former, consistent, and correct snapshot of your data lake.
+* **Enforce cross collection consistency** - provide to consumers several collections of data that must be synchronized, in one atomic, revertable, action.
```

Comment:
Please undo this change "revertible" -> "revertable".  What happened was a concurrent PR fixed this spelling and now your PR is changing it back to the previous incorrect version.